internal/gen3fuse.go

```diff
@@ -132,8 +132,13 @@ func InitializeInodes(didToFileInfo map[string]*indexdResponse) map[fuseops.Inod
 	
 	k := 0
 	for did, fileInfo := range didToFileInfo {
-		var filename = fileInfo.Filename  // TODO: Determine whether the true filename is preferable, or just the DID
-
+		if fileInfo.Filename == "" {
+			FuseLog(fmt.Sprintf("Indexd record %s does not seem to have a file associated with it; ignoring it.", did))
```

Comment:
checking whether a record has file associated it should be done by checking the record.urls is empty or not